testnets/xarchaintestnet/assetlist.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "../assetlist.schema.json",
```

Comment:
for testnets, the schema reference must seek up another directory
should be 
`"$schema": "../../assetlist.schema.json",`
(and similarly in the chain.json and versions.json)

Reply:
fixed the schema path